src/Arcus.Scripting.Storage.Table/Scripts/Set-AzTableStorageEntities.ps1

```diff
@@ -8,6 +8,12 @@ param(
 if (-not (Test-Path -Path $ConfigurationFile)) {
     throw "Cannot re-create entities based on JSON configuration file because no file was found at: '$ConfigurationFile'"
 }
+if ((Get-Content -Path $ConfigurationFile -Raw) -eq $null) {
+     throw "Cannot re-create entities based on JSON configuration file because the file is empty."
+}
+if (-not (Get-Content -Path $ConfigurationFile -Raw | Test-Json)) {
```

Comment:
Maybe also check that it is an array? Or fallback to an with a single item if it is a JSON object instead of an JSON array?

Reply:
Added a JSON schema to make sure the config file is correct, see [`5e586e9` (#222)](https://github.com/arcus-azure/arcus.scripting/pull/222/commits/5e586e9e96ade83d94d129048242762572471275)